### ISSUE TITLE
Why application needs permissions to photo/media files?

### ISSUE THREAD

**Issue by webcapcha** [thuryn/your-local-weather#129]
The title of question is clear enough.

**Comment by Locash** [comment]
So the debug option can save the logs to a folder of choice.

**Comment by webcapcha** [comment]
Okay, why that application need to know my phone number? (it also ask that permission)

**Comment by Locash** [comment]
@webcapcha 
Google location services use wifi info(no active connection needed but just discoverable), along with cell tower;  then compares that to their respective mapping data to acquire a users location without using Gps; which is a resource hog. The other two are passively collected from resources already in use by most devices. Gps is also used but requires a service also so the coordinates received can be reversed geocoded into a named location.

The local mls(mozilla location service) and nominatim option use the same means BUT do not use personally identifiable info(ie. an google account) to request such data. This option is selected by default. 
You are able to turn off request location and just enter a location manually .



![Untitled19](https://user-images.githubusercontent.com/30946440/64752323-c688d900-d4e4-11e9-8259-87e3623acc90.png)

**Comment by webcapcha** [comment]
@Locash you are wrong. I'm asking why app needs permission to:

> read phone status and identity
> Allows the app to access the phone features of the device. This permission allows the app to determine the phone number and device IDs, whether a call is active, and the remote number connected by a call.

That permission doesn't needed for determine location. Please read Android docs.

**Comment by thuryn** [comment]
@webcache Yes, the permission is needed to get GPS cells, @Locash is right.

**Comment by thuryn** [comment]
@Locash thank you for the great explanation. It's not good, that your location is not save, but it's fact. On the other hand, sharing the location (for example in Play) is because of a legislation - so it's not fault of Google.

**Comment by iillyyaa2033** [comment]
Should this issue be closed now?
